packages/ckeditor5-dev-release-tools/lib/utils/executeinparallel.js

```diff
@@ -48,6 +48,7 @@ export default async function executeInParallel( options ) {
 		concurrency = os.cpus().length / 2
 	} = options;
 
+	const concurrencyAsInteger = Math.floor( concurrency ) || 1;
```

Comment:
We need a few tests:
* `concurrency ` is a float number (specified as an argument)
* `concurrency ` is a float number (defined as a default value, `os.cpus()` returns an odd number)
* `concurrency ` is an integer equal to `0`

I would skip negative numbers.

Reply:
Added in [c6ffb35](https://github.com/ckeditor/ckeditor5-dev/pull/1015/commits/c6ffb35970cbdb01d04dff7f40665990f72fe160)